### PR TITLE
#1958 Bootstrap Knowledge graph service in backend

### PR DESCRIPTION
**Pull request by stephanevieira75**
# TODOS: 

- [ ] publish onChannelCreated
- [ ] publish onWorkspaceCreated
- [ ] publish onUserCreated
- [ ] publish onCompanyCreated
- [ ] publish onMessageCreated
- [ ] Send data using KnowledgeGraphAPIClient
- [ ] Fix apiUrl in KnowledgeGraphAPIClient

### CODE REVIEW

**Comment by RomaricMourgues** [comment]
- To fix the error add a test like if this endpoint is not defined (empty string) then we don't send anything to the knowledge graph service. The default is to have an empty string in config/default by the way.
- Could we add an option in config to only send to the knowledge graph data coming from a specific company ?

File: twake/backend/node/config/default.json [hunk 1]
```diff
@@ -147,7 +147,8 @@
   },
   "knowledge-graph": {
     "endpoint": "http://host-gateway:8888",
-    "active": false
+    "use": false,
+    "company_idx": []
```
Comment (stephanevieira75) [inline twake/backend/node/config/default.json:151]: Maybe we should rename this as `allowed_companies`.

Comment (stephanevieira75) [reply twake/backend/node/config/default.json]: Done 👍🏻

**Comment by RomaricMourgues** [comment]
@stephanevieira75 tests are not passing, something is not initiated somehow here is the interesting log:
```
2022-05-03T15:46:29.5879925Z 
2022-05-03T15:46:29.7332697Z (node:312) UnhandledPromiseRejectionWarning: TypeError: Cannot read properties of undefined (reading 'get')
2022-05-03T15:46:29.7333502Z     at KnowledgeGraphService.getKnowledgeGraphApiClient (/usr/src/app/src/core/platform/services/knowledge-graph/index.ts:111:41)
2022-05-03T15:46:29.7334173Z     at new KnowledgeGraphService (/usr/src/app/src/core/platform/services/knowledge-graph/index.ts:26:57)
2022-05-03T15:46:29.7334706Z     at GlobalResolver.doInit (/usr/src/app/src/services/global-resolver.ts:205:29)
2022-05-03T15:46:29.7335093Z     at processTicksAndRejections (node:internal/process/task_queues:96:5)
2022-05-03T15:46:29.7335433Z     at init (/usr/src/app/test/e2e/setup/index.ts:60:5)
2022-05-03T15:46:29.7335843Z     at Object.<anonymous> (/usr/src/app/test/e2e/counters.spec.ts:36:16)
2022-05-03T15:46:29.7336290Z (Use `node --trace-warnings ...` to show where the warning was created)
2022-05-03T15:46:29.7336476Z 
2022-05-03T15:46:29.7351385Z (node:312) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
2022-05-03T15:46:29.7352132Z 
```

**Comment by RomaricMourgues** [comment]
Please check
1. Service can work if configuration isn't complete
2. Test configuration file is ok
